Examples/Tests/reduced_diagnostics_python/PICMI_inputs_reduced.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+import numpy as np
+from pywarpx import picmi
+
+# Physical constants
+c = picmi.constants.c
+
+max_steps = 200
+
+# Create grid
+grid = picmi.Cartesian3DGrid(
+    number_of_cells=[32, 32, 32],
+    lower_bound=[-1., -1., -1.],
+    upper_bound=[1., 1., 1.],
+    lower_boundary_conditions=['periodic', 'periodic','periodic'],
+    upper_boundary_conditions=['periodic', 'periodic','periodic'],
+    lower_boundary_conditions_particles=['periodic', 'periodic','periodic'],
+    upper_boundary_conditions_particles=['periodic', 'periodic','periodic'],
+    warpx_max_grid_size=32)
+
+electrons = picmi.Species(
+        particle_type='electron',
+        name=f'electrons',
+        initial_distribution= picmi.UniformDistribution(
+                                density=1.e14,
+                                rms_velocity = [0.035*c]*3
+        ))
+photons = picmi.Species(
+        particle_type='photon',
+        name='photons',
+        initial_distribution=picmi.UniformDistribution(
+                                density=1.e14,
+                                rms_velocity = [0.2*c]*3)
+)
+
+protons = picmi.Species(
+        particle_type='proton',
+        name=f'protons',
+        initial_distribution=picmi.UniformDistribution(
+                                density=1.e14
+        ))
+
+species_list = [electrons, protons, photons]
+
+
+# reduced diagnostics
+EP = picmi.ReducedDiagnostic(name='EP', type='ParticleEnergy',intervals=200)
+NP = picmi.ReducedDiagnostic(name='NP', type='ParticleNumber',intervals=200)
+EF = picmi.ReducedDiagnostic(name='EF', type='FieldEnergy',intervals=200)
+PP = picmi.ReducedDiagnostic(name='PP', type='ParticleMomentum',intervals=200)
+PF = picmi.ReducedDiagnostic(name='PF', type='FieldMomentum',intervals=200)
+MF = picmi.ReducedDiagnostic(name='MF', type='FieldMaximum',intervals=200)
+MR = picmi.ReducedDiagnostic(name='MR', type='RhoMaximum',intervals=200)
+FP = picmi.ReducedDiagnostic(name='FP',
+                                type='FieldProbe',
```

Comment:
My preference is that arguments on separate lines should line up with the start of the first argument since it looks cleaner. This is just a suggestion and is at your own discretion.

Reply:
it does look cleaner.  Done!